sup3r/pipeline/forward_pass.py

```diff
@@ -755,7 +754,7 @@ def __init__(self, strategy, node_index=0):
             logger.error(msg)
             raise KeyError(msg)
 
-        self.model = model_class.load(*self.model_args)
+        self.model = model_class.load(*self.model_args, verbose=False)
```

Comment:
I think the first model load in the non-parallel section should print the model versions (verbose=True)

Reply:
Yeah good point